draw_ball.py

```diff
@@ -1,12 +1,9 @@
 from num_gen import generator
 
+
 def ball(n):
     numbers = []
-    while len(numbers) <= n:
+    while len(numbers) < n:
         gen = generator(1, 35)
-        numbers.insert(n, gen)
-
+        numbers.append(gen)
```

Comment:
I think that there is no need for a while loop here. You can use a simple for loop. and you can use list comprehension to populate the numbers list